client/packages/system/src/Item/context/usePackVariant.ts

```diff
@@ -69,6 +69,12 @@ export const useRefreshPackVariant = () => {
   }, [data, setItems]);
 };
 
+export const usePackVariantsEnabled = () => {
```

Comment:
very neat approach 😍 
Just wondering how we can switch on / off for demo / gathering feedback.. there's no system level switching, so we have to enable, then configure, then unconfigure if we are using a demo server to get feedback from PMs for example.